### ISSUE TITLE
version choice question

### ISSUE THREAD

**Issue by acewhite1010** [acewhite1010/aa-starting-off-discussion#2]
### A: Choose a version to run on. 

Versions: 1.6 and 1.11+ (one category for each version from there)

There are multiple versions to play, and they mostly differ a lot in terms of time dedicated and strategies. Most common version to run on is 1.16.1, for its popularity in the any% community, and its development in strats and game. Definitely a good choice if you play on the later versions.

Though, if you mainly play on the older versions, or a old-Minecraft fan, you can try 1.6, as it's the easiest to play, has the least achievements, and in average least time needed (might not include resetting). There's also 1.11 ~ 1.13, which is another different spectrum of running AA.

TLDR choose the one version you play the most, or 1.16.1 if you're clueless. Or try them out yourselves.
